hermes/src/aggregate.rs

```diff
@@ -333,7 +334,7 @@ where
         .iter()
         .map(|message_state| match message_state.message {
             Message::PriceFeedMessage(price_feed) => Ok(PriceFeedUpdate {
-                price_feed:  PriceFeed::new(
+                price_feed:        PriceFeed::new(
```

Comment:
the extra spacing seems weird

Reply:
yeah it is the rust formatter config that we have :D